project/plugins.sbt

```diff
@@ -1,15 +1,15 @@
 scalacOptions += "-deprecation"
 libraryDependencies += "org.slf4j" % "slf4j-nop" % "1.7.25"
 
-addSbtPlugin("com.typesafe"                      % "sbt-mima-plugin"       % "0.1.14")
+addSbtPlugin("com.typesafe"                      % "sbt-mima-plugin"       % "0.1.18")
 addSbtPlugin("com.typesafe.sbt"                  % "sbt-osgi"              % "0.9.2")
 addSbtPlugin("com.eed3si9n"                      % "sbt-buildinfo"         % "0.7.0")
-addSbtPlugin("com.typesafe.sbt"                  % "sbt-git"               % "0.9.2")
+addSbtPlugin("com.typesafe.sbt"                  % "sbt-git"               % "0.9.3")
 addSbtPlugin("org.scala-js"                      % "sbt-scalajs"           % "0.6.21")
-addSbtPlugin("com.github.gseitz"                 % "sbt-release"           % "1.0.4")
-addSbtPlugin("com.jsuereth"                      % "sbt-pgp"               % "1.0.1")
-addSbtPlugin("org.xerial.sbt"                    % "sbt-sonatype"          % "1.1")
-addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings"      % "0.2.2")
-addSbtPlugin("org.scoverage"                     % "sbt-scoverage"         % "1.5.0")
+addSbtPlugin("com.github.gseitz"                 % "sbt-release"           % "1.0.6")
```

Comment:
sbt-release 1.0.6 does not work with sbt 1.1.0. please update to `1.0.7`
- https://github.com/sbt/sbt/issues/3792
- https://github.com/sbt/sbt-release/commit/66728b59bbf9d9463b69928bc16aee514c94f1d0